src/codewhispererChat/controllers/chat/telemetryHelper.ts

```diff
@@ -359,21 +389,19 @@ export class CWCTelemetryHelper {
         return Math.round(chunkTimes[1] - chunkTimes[0])
     }
 
-    // private getResponseStreamTimeBetweenChunks(tabID: string): string {
-    //     try {
-    //         const chunkDeltaTimes: number[] = []
-    //         const chunkTimes = this.responseStreamTimeForChunks.get(tabID) ?? [0]
-    //         for (let idx = 0; idx < chunkTimes.length - 1; idx++) {
-    //             chunkDeltaTimes.push(Math.round(chunkTimes[idx + 1] - chunkTimes[idx]))
-    //         }
-
-    //         const trimmed = JSON.stringify(chunkDeltaTimes).slice(0, 2_000)
-    //         const fixed = trimmed.endsWith(',') ? trimmed.slice(0, -1) : trimmed
-    //         return fixed.endsWith(']') ? fixed : `${fixed}]`
-    //     } catch (e) {
-    //         return '[-1]'
-    //     }
-    // }
+    private getResponseStreamTimeBetweenChunks(tabID: string): number[] {
```

Comment:
Can you add doc string to explain why return [-1] at catch(e)? Also can you add log when catching an exception?

Reply:
Done. Added log. Returning empty array now. `[-1]` was an artifact of previous code changes.